Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -184,6 +185,12 @@ A playbook that given the IP address enriches GCP information relevant to ASM al
 
 ![Cortex ASM - GCP Enrichment](https://github.com/demisto/content/raw/234817ee736286eee1b40ece897cb7b2974771dc/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_GCP_Enrichment.png)
 
+#### Cortex ASM - Instant Message
+
+A playbook that is used to create instant messages toward service owners to notify them of their internet exposures.
+
+![ortex ASM - Instant Message](https://raw.githubusercontent.com/demisto/content/f00e2443782383c76dcfa7e537c03940afe3197c/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Instant_Message.png)
```

Comment:
I think C is missing here

Reply:
Fixed in https://github.com/demisto/content/pull/34827/commits/6aebe6d769db73660d6872e1ff3233f786efce23

good catch!